okio/src/commonMain/kotlin/okio/internal/Buffer.kt

```diff
@@ -40,7 +40,7 @@ private val DIGITS = "0123456789abcdef".asUtf8ToByteArray()
  * Returns true if the range within this buffer starting at `segmentPos` in `segment` is equal to
  * `bytes[bytesOffset..bytesLimit)`.
  */
-internal fun rangeEquals(
+internal inline fun rangeEquals(
```

Comment:
I went through and marked everything as `inline`. Not sure it really makes sense for a couple of these methods (like this one) so let me know if I should remove it from any of these methods.

Reply:
cool. I think our rule should be “an inlined function shouldn’t be more than 5 lines or be used more than 2x”

Reply:
... or 10 lines, 3 times, or something. But a firm rule that’s easy to follow.

Reply:
So I should remove `inline` from the non-commonX methods here unless they are a higher-order function?